scripts/run_e2e_tests.py

```diff
@@ -240,6 +240,20 @@ def update_dev_mode_in_constants_js(constant_file, dev_mode_setting):
     common.inplace_replace_file(constant_file, pattern, replace)
 
 
+def update_show_classroom_in_constants(constant_file, show_classroom_setting):
```

Comment:
Do not do this please. Use regular methods and a flag in the config dashboard instead.

Reply:
Hey @seanlip, I tried replacing this constant with a config property and then changing the value before and after running the e2e tests. The config property behaves as expected, all the tests (frontend and backend) are passing locally, the only issue is importing config_domain in the run_e2e_tests.py file is throwing some errors. I tried resolving some of them but I can't seem to resolve this one:
<details>
  <summary>Error:</summary>

```
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/prayush/Desktop/opensource/oppia/scripts/run_e2e_tests.py", line 477, in <module>
    main()
  File "/home/prayush/Desktop/opensource/oppia/scripts/run_e2e_tests.py", line 450, in main
    config_property.set_value('committer_id', True)
  File "core/domain/config_domain.py", line 213, in set_value
    self.name, strict=False)
  File "core/storage/base_model/gae_models.py", line 1003, in get
    return super(VersionedModel, cls).get(entity_id, strict=strict)
  File "core/storage/base_model/gae_models.py", line 196, in get
    entity = cls.get_by_id(entity_id)
  File "/home/prayush/Desktop/opensource/oppia_tools/google_appengine_1.9.67/google_appengine/google/appengine/ext/ndb/utils.py", line 160, in positional_wrapper
    return wrapped(*args, **kwds)
  File "/home/prayush/Desktop/opensource/oppia_tools/google_appengine_1.9.67/google_appengine/google/appengine/ext/ndb/model.py", line 3609, in _get_by_id
    return cls._get_by_id_async(id, parent=parent, **ctx_options).get_result()
  File "/home/prayush/Desktop/opensource/oppia_tools/google_appengine_1.9.67/google_appengine/google/appengine/ext/ndb/tasklets.py", line 383, in get_result
    self.check_success()
  File "/home/prayush/Desktop/opensource/oppia_tools/google_appengine_1.9.67/google_appengine/google/appengine/ext/ndb/tasklets.py", line 427, in _help_tasklet_along
    value = gen.throw(exc.__class__, exc, tb)
  File "/home/prayush/Desktop/opensource/oppia_tools/google_appengine_1.9.67/google_appengine/google/appengine/ext/ndb/context.py", line 722, in get
    deadline=memcache_deadline)
  File "/home/prayush/Desktop/opensource/oppia_tools/google_appengine_1.9.67/google_appengine/google/appengine/ext/ndb/tasklets.py", line 430, in _help_tasklet_along
    value = gen.send(val)
  File "/home/prayush/Desktop/opensource/oppia_tools/google_appengine_1.9.67/google_appengine/google/appengine/ext/ndb/context.py", line 1104, in _memcache_get_tasklet
    rpc = memcache.create_rpc(deadline=deadline)
  File "/home/prayush/Desktop/opensource/oppia_tools/google_appengine_1.9.67/google_appengine/google/appengine/api/memcache/__init__.py", line 297, in create_rpc
    return apiproxy_stub_map.UserRPC('memcache', deadline, callback)
  File "/home/prayush/Desktop/opensource/oppia_tools/google_appengine_1.9.67/google_appengine/google/appengine/api/apiproxy_stub_map.py", line 414, in __init__
    self.__rpc = CreateRPC(service, stubmap)
  File "/home/prayush/Desktop/opensource/oppia_tools/google_appengine_1.9.67/google_appengine/google/appengine/api/apiproxy_stub_map.py", line 68, in CreateRPC
    assert stub, 'No api proxy found for service "%s"' % service
AssertionError: No api proxy found for service "memcache"
```
</details>

Also, the overall code changes are getting unnecessarily complicated with this method. @kevintab95 suggested that we could add a button in the admin panel that could toggle the constant and then manually make the change through the e2e tests while running them. That seems a simpler and safer alternative, so should I implement that instead?

Reply:
@iamprayush, You can create a config property CLASSROOM_PAGE_ENABLED [here](https://github.com/oppia/oppia/blob/develop/core/domain/config_domain.py#L300) with a default value False and enable it through the config tab of admin page. [make sure to update new structure team about this change once this PR is merged]